### DISCUSSION TITLE
Show table data by user

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#6976] [Q&A]
I would like to confirm whether it is currently possible to configure the system so that different users can see different table data. Specifically, after personnel submit data through a form, the administrator can view the form data submitted by all personnel. However, when different individuals log into the system, they should only be able to see the data they have submitted and not the data submitted by others. This would enhance data privacy. Is this functionality supported now?

**Comment by dstala** [comment]
@c2006878 You are requesting a feature where "Only the user who had created a particular record should be able to edit it". This feature is currently not supported.

**Reply by c2006878** [reply]
I think this feature is important. Sometimes we don't want to see what others have, but rather have our own library of achievements.
